features/step_eight.feature

```diff
@@ -59,7 +59,7 @@ Feature: Step eight page
   Scenario: No income
     Given I am a single person on step eight
     When I select no income
-    Then I am taken to step eleven
+    Then I am taken to step '11''probate'
```

Comment:
I think we can remove these single quotes
`Then I am taken to step 11 probate`

or at least the ones in the middle
`Then I am taken to step '11 probate'`